logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserRepository.kt

```diff
@@ -67,7 +70,10 @@ class UserDataSource(
             }.coFold({
                 Either.Left(it)
             }, {
-                userDAO.insertUsers(it.map(userMapper::fromApiModelToDaoModel))
+                val usersToBePersisted = it.map(userMapper::fromApiModelToDaoModel)
+                // Save (in case there is no data) a reference to the asset id (profile picture)
+                assetRepository.saveUserPictureAsset(mapAssetsForUsersToBePersisted(usersToBePersisted))
```

Comment:
I think you can delete this. We don't need to insert an empty asset.

Reply:
Oh, yes absolutely ! I took care of this in the next PR. You could see more at https://github.com/wireapp/kalium/pull/243/commits/eee3ed0726b6983307dedbede2f740244d43f892#diff-f768a46d2d5299842bd48f5a3c654e69fb0ed6712d749b4e82925eebecfeb709L55 #243

Reply:
👍 saw that later